traitsui/tests/editors/test_list_editor.py

```diff
@@ -170,6 +179,35 @@ def test_index_out_of_range(self):
             with self.assertRaises(IndexError):
                 people_list.locate(Index(10))
 
+    # regression test for enthought/traitsui#1154
+    @requires_toolkit([ToolkitName.qt])
+    @unittest.skipIf(no_modal_dialog_tester, "ModalDialogTester unavailable")
```

Comment:
I'm not entirely sure why the modal dialog tester would be unavailable with the qt toolkit.

Reply:
AFAIK It wouldn't be, I can remove this